### PR TITLE
Fix icon imports for commission members

### PR DESCRIPTION
**Pull request by Fastjur**

### CODE REVIEW

**Comment by Fastjur** [comment]
@skamoen @martijnjanssen 

Decide what you want. I'm fucking done with this one

**Comment by elarb** [comment]
Instead of having to provide both the set and the icon name, I would propose only sending the icon name from the API, e.g. `icons:gavel` -> `gavel` and always use our own set with only icons we need.

File: src/lancie-commission/lancie-commission-member.html [hunk 1]
```diff
@@ -1,12 +1,8 @@
 <link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
-<link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
```
Comment (TimvdLippe) [inline src/lancie-commission/lancie-commission-member.html]: Um, should this be removed? We are still using iron-icon.

Comment (elarb) [reply src/lancie-commission/lancie-commission-member.html]: Good catch, I was getting a little too excited removing lines from the code 😄

**Comment by elarb** [comment]
Can I have a confirmation before merging this?

**Comment by TimvdLippe** [comment]
Does this work without modifying the API? Do we still want to change the API format? If so please file an issue on that repo

**Comment by elarb** [comment]
The API is compatible

**Comment by TimvdLippe** [comment]
It will break when we publish live though, so we have to take that into mind

@skamoen @martijnjanssen @MatthijsKok When we publish live, we have to remind ourselves to update these values to the ones in the testdatarunner.

**Pull request merged**